src/modules/dashboard/components/ColonyEvents/ColonyEventsListItem.tsx

```diff
@@ -108,6 +112,8 @@ const ColonyEventsListItem = ({
     metadata,
     tokenAddress,
     paymentId,
+    fromPot: displayValuesObject.fromPot,
+    toPot: displayValuesObject.toPot,
```

Comment:
This shouldn't be needed anymore once you remove the `filtereEvents` logic, as it should just display data that comes in from the event only